polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -4859,8 +4900,10 @@ mlir::func::FuncOp MLIRASTConsumer::GetOrCreateMLIRFunction(
   if (!FD->isDefined(Def, /*checkforfriend*/ true))
     Def = FD;
 
-  if (functions.find(name) != functions.end()) {
-    auto function = functions[name];
+  auto &Context = F.context;
```

Comment:
auto -> MLIRContext

Reply:
It was a `FunctionContext` in fact. Changed.